spec/features/avatar_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'rails_helper'
```

Comment:
Tomorrow, ask me to explain how utilizing a FactoryGirl.build(blahblahblah would be helpful to refactor this test 👍